### PR TITLE
fix: Some wrong logging for flattened updates

### PR DESCRIPTION
**Pull request by hasanawad94**
## Changes:
Now the filter in the log message is also filtering strings having only white spaces

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/workers/repository/updates/branchify.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export async function branchifyUpgrades(
   logger.debug(
     `${updates.length} flattened updates found: ${updates
       .map((u) => u.depName)
-      .filter((txt) => txt?.length)
+      .filter((txt) => txt?.length && txt.trim().length)
```
Comment (viceice) [inline lib/workers/repository/updates/branchify.ts:25]:
```suggestion
      .filter((txt) => txt?.trim().length)
```

simpler 😉

**Comment by hasanawad94** [comment]
This pr fixes: https://github.com/renovatebot/renovate/issues/6134

**Comment by rarkins** [comment]
Strange if we end up with depNames with nothing but white space though?

**Review by rarkins — APPROVED** [review]

**Comment by hasanawad94** [comment]
> Strange if we end up with depNames with nothing but white space though?

@rarkins Yes I agree . I've tried to investigate on why empty strings were being taken by using this repo here: https://github.com/renovate-tests/hops.
But I've gotten errors in the configuration saying : `Message: packageRules[2]: packageRules cannot combine both matchUpdateTypes and rangeStrategy. Rule: {"rangeStrategy":"bump","matchDepTypes":["devDependencies"],"matchUpdateTypes":["major","minor"]}`

**Comment by rarkins** [comment]
That error message is correct

**Pull request merged**